### DISCUSSION TITLE
Search bar view height

### DISCUSSION THREAD

**Discussion by Dave-d-1st** [flet-dev/flet#3983] [Q&A]
### Question

Is there a way to change the view height of a search bar as to fit its contents in a specific way because it looks kind of weird when there's only few items in the view and the entire view height is being displayed

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
https://github.com/flet-dev/flet/issues/3985
